ReactWindows/ReactNative/Modules/Clipboard/IClipboardInstance.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Windows.ApplicationModel.DataTransfer;
+
+namespace ReactNative.Modules.Clipboard
+{
+    /// <summary>
+    /// Gets and sets information from the clipboard object.
+    /// </summary>
+    public interface IClipboardInstance
+    {
+        /// <summary>
+        /// Removes all data from the Clipboard.
+        /// </summary>
+        void Clear();
+
+        /// <summary>
+        /// Gets the current content that is stored in the clipboard object.
+        /// </summary>
+        /// <returns>Contains the content of the Clipboard.</returns>
+        DataPackageView GetContent();
```

Comment:
You're probably already aware of this but DataPackageView is sealed with an internal constructor, while will make it quite difficult to mock.